tasks/build-path-robustness.md

```
@@ -0,0 +1,140 @@
+# Build-path robustness refactor
+
+## Problem
+
+FastLED reported a build artifact path like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)